CelesteNet.Server.FrontendModule/Content/frontend/cp/js/components/dialog.js

```diff
@@ -35,9 +40,15 @@ export class FrontendDialog {
 
       return el;
     }
-    const id = (id, gen) => el => {
+    const input = (id, gen, formid) => el => {
```

Comment:
Missing space between functions.

Reply:
Not sure I interpreted this correctly but I put blank lines between the function-variable-declarations? (idk what exactly to call them)